playground_v2/src/lib/mamaLanguage.js

```diff
@@ -0,0 +1,55 @@
+const convertToJS = (sourceCode) => {
```

Comment:
This is a great idea. Now lets take it a step further can you import this into the cli.js?

So we only have one part where we have all the translations for playground and the command line?

Reply:
Thanks for the suggestion @ahnafyy. It's a great joy to work on this repo! I have made some modifications as you instructed! Check them. I hope it's ready to be merged!